app/lib/common/widgets/acter_icon_picker/model/color_data.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+List<Color> iconPickerColors = [
```

Comment:
This contains set of ActerIcons Colors.

![Screenshot 2024-09-05 at 1 54 51 PM](https://github.com/user-attachments/assets/63da8d3d-df50-446e-986f-2a994c29009f)